src/RobotModelUpdate.cpp

```diff
@@ -36,20 +47,123 @@ void RobotModelUpdate::reset(mc_control::MCGlobalController & controller)
   mc_rtc::log::info("Robot Update is:\n{}", robotUpdate.dump(true, true));
 
   gui.removeElements(this);
-  gui.addElement(this,
-      {"RobotModelUpdate", robot_},
-      mc_rtc::gui::Button("Reset to default",
-        [this, &ctl]()
-        {
-          resetToDefault(ctl.robot(robot_));
-        })
-  );
-
-  robotUpdate.addToGUI(gui, {"RobotModelUpdate", robot_}, "Update Robot Model", 
-       [this, &ctl]() {
-        mc_rtc::log::info("Updated robot schema");
-        updateRobotModel(ctl);
-       });
+  gui.addElement(this, {"Plugin", "RobotModelUpdate", robot_},
+                 mc_rtc::gui::Button("Reset to default",
+                                     [this, &ctl]()
+                                     {
+                                       resetToDefault(ctl.robot(robot_));
+                                       resetToDefault(ctl.outputRobot(robot_));
+                                     }),
+                 mc_rtc::gui::Button("Load Xsens config", [this, &ctl]() { configFromXsens(ctl); }));
+
+  robotUpdate.addToGUI(gui, {"Plugin", "RobotModelUpdate", robot_}, "Update Robot Model",
+                       [this, &ctl]()
+                       {
+                         mc_rtc::log::info("Updated robot schema");
+                         updateRobotModel(ctl);
+                       });
+}
+
+void RobotModelUpdate::configFromXsens(mc_control::MCController & ctl)
+{
+  // New version: using data directly from xsens to scale
+  // step 1: get poses (world frame)
+  auto Hips_W_0 = ctl.datastore()
+                      .call<sva::PTransformd>("XsensPlugin::GetSegmentPose", static_cast<const std::string &>("Pelvis"))
```

Comment:
Small note on notation. In general with `SpaceVecAlg` the convention is to write frame names as `X_A_B` which means a transform from `A` to `B`. By convention we write `0` as the world frame. So the notation here does not make much sense as `_W_0` would mean from "world to world". Using more standard notations you'd want to name it `X_0_Hips`.

The main advantage of keeping with this consistent notation is that it makes it:
1. Clear which frames are involved
2. Easier to work with. For instance `X_A_C = X_B_C * X_A_B` which you can write mechanically and you're sure to be correct.